react/hooks/useSelectedFilters.js

```diff
@@ -1,15 +1,12 @@
 import { zip } from 'ramda'
-import { useContext } from 'react'
-
-import QueryContext from '../components/QueryContext'
 
 /**
  * This hook is required because we make the facets query
  * with only the categories and fulltext parameters, so we
  * need to calculate manually if the other filters are selected
  */
-const useSelectedFilters = facets => {
-  const { query, map } = useContext(QueryContext)
+const useSelectedFilters = (facets, queryArgs) => {
+  const { query, map } = queryArgs
```

Comment:
the usage of a context is because this data is used everywhere and in a lot of places and so we don't  want to pass this everywhere. Why the need for queryArgs, can't you set the appropriate value in context? That would be the correct